workflow/react-project/src/components/common/AuthenticationNotification/index.jsx

```diff
@@ -0,0 +1,12 @@
+import './style.less';
+
+const AuthenticationNotification = ({ normalText, coloredText, onClick }) => (
+  <p className="sign-up-notification">
+    {normalText}
+    <span className="sign-up-text" onClick={onClick}>
+      {coloredText}
+    </span>
+  </p>
+);
+
+export default AuthenticationNotification;
```

Comment:
`AuthenticationNotification` is not common component
Common component is `Notification` which gets type (is it error or success) and message

Rewrite it, change classes (they have to be common, without `sign-up`)